### PR TITLE
Fix me

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Me is broke, this fix me. This is a me problem cause by me, so me fixed it. 

More seriously, me is an emote with the key ="me", and it was a duplicate key with the keybind me. This broke me, because only the latter keybind (the emote, that shouldn't be called directly) was showed on screen

## Changelog
:cl:
fix: Me is fixed by me cause it was broken by me.
/:cl:

### CODE REVIEW

**Comment by usnpeepoo** [comment]
u need fixing?

**Comment by TiviPlus** [comment]
can you explain how it fixes it thnx

**Comment by SplinterGP** [comment]
i wont fix you

File: code/modules/mob/emote.dm [hunk 1]
```diff
@@ -51,9 +51,10 @@
 
 
 /datum/emote/custom
-	key = "me"
+	key = "emote_me"
```
Comment (TiviPlus) [inline code/modules/mob/emote.dm:54]: Explain this change?

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**